octopi.js

```diff
@@ -35,4 +43,29 @@ Octopi.prototype = {
       return tbl[id];
     });
   }
+  /**
+     * Serialize the Octopi trie as string
+     *
+     * @return String
+     */
+  serialize: function(){
+    
+    var o = new Object();
```

Comment:
Well I did the modifications. I am going to commit!
Do you want me to do another pull request?

-- 
Celine Bursztein
------------------------------------------------------
http://celine.im
@cealtea <https://twitter.com/cealtea> on Twitter, +Celine Bursztein
<https://plus.google.com/+Celine%20Bursztein> on Google+

On Mon, Sep 5, 2016 at 6:49 PM Eeo Jun <notifications@github.com> wrote:

> In octopi.js
> <https://github.com/eugene-eeo/octopi/pull/2#discussion_r77563960>:
>
> > @@ -35,4 +43,29 @@ Octopi.prototype = {
> >        return tbl[id];
> >      });
> >    }
> > +  /**
> > +     * Serialize the Octopi trie as string
> > +     *
> > +     * @return String
> > +     */
> > +  serialize: function(){
> > +
> > +    var o = new Object();
>
> Sure, are you working on it or should I merge + fix?
>
> —
> You are receiving this because you were mentioned.
> Reply to this email directly, view it on GitHub
> <https://github.com/eugene-eeo/octopi/pull/2/files/04687289913bb58d39c468eada293fa7926d5efa#r77563960>,
> or mute the thread
> <https://github.com/notifications/unsubscribe-auth/ABICDt63vhJ0l2jFK82vVi__J_RsDqEWks5qnMavgaJpZM4J1TFC>
> .
>